src/v2/guide/custom-directive.md

```diff
@@ -143,37 +143,71 @@ new Vue({
 </script>
 {% endraw %}
 
-Directive arguments can be dynamic. For example, in `v-mydirective:argument=[dataproperty]`, `argument` is the string value assigned to the *arg* property in your directive hook *binding* parameter and `dataproperty` is a reference to a data property on your component instance assigned to the *value* property in the same *binding* parameter. As directive hooks are invoked, the *value* property of the *binding* parameter will dynamically change based on the value of `dataproperty`.
+### Dynamic Directive Arguments
 
-An example of a custom directive using a dynamic argument:
+Directive arguments can be dynamic. For example, in `v-mydirective:[argument]="value"`, the `argument` can be updated based on data properties in our component instance! This makes our custom directives flexible for use throughout our application.
+
+Let's say you want to make a custom directive that allows you to pin elements to your page using fixed positioning. We could create a custom directive where the value updates the vertical positioning in pixels, like this:
 
 ```html
-<div id="app">
+<div id="baseexample">
   <p>Scroll down the page</p>
-  <p v-tack:left="[dynamicleft]">I’ll now be offset from the left instead of the top</p>
+  <p v-pin="200">Stick me 200px from the top of the page</p>
+</div>
+```
+
+```js
+Vue.directive('pin', {
+  bind(el, binding, vnode) {
```

Comment:
Sorry, missed this in my previous review. Shouldn’t we use ES5 syntax per our doc convention?

Reply:
Ah, yep. I think I left a const in there as well 😅